sys-apps/onerng/onerng-3.6.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit distutils-r1 udev
+
+DESCRIPTION="Software for the Open Hardware Random Number Generator called OneRNG"
+HOMEPAGE="http://www.onerng.info/"
+SRC_URI="https://github.com/OneRNG/onerng.github.io/raw/master/sw/${PN}_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+RDEPEND="app-crypt/gnupg
+	dev-python/python-gnupg[${PYTHON_USEDEP}]
+	sys-apps/rng-tools
+	sys-process/at
+	virtual/udev"
+
+DEPEND="virtual/pkgconfig
+	virtual/udev"
+
+S="${WORKDIR}/${PN}_${PV}"
+
+src_configure() {
+	# Run configure
+	econf
+}
+
+src_compile() {
+	# Run make
+	emake
+}
```

Comment:
why?

Reply:
This is due "inherit distutils-r1". Without it, ${PYTHON_USEDEP} seems not to work.
I've set this, to get the right dependencies for python.

The package onerng is mainly written in python, but uses a standard makefile.

Due "inherit distutils-r1", portage now tries to build this as a python package, thus this will fail.
-> can't open file 'setup.py': [Errno 2] No such file or directory

That's why I have definied src_configure and src_compile.

Is there any better way?